.gitignore

```diff
@@ -86,3 +86,4 @@ examples/generated/test/LonaViewer/LonaViewer.xcodeproj/project.xcworkspace/xcsh
 examples/LonaViewer/LonaViewer.xcodeproj/project.xcworkspace/contents.xcworkspacedata
 
 .vscode
+node_modules
```

Comment:
Something in vscode keeps creating this directory for me, with a debug log. So that's why i added it